### ISSUE TITLE
.exe file support

### ISSUE THREAD

**Issue by thebeeper2524** [chrisj42/minicraft-plus-revived#167]
Can ya add .exe support to this game? I'd be delighted if ya did that! _isnt it weird that it only is a .jar file?_

**Comment by thebeeper2524** [comment]
> Can ya add .exe support to this game? I'd be delighted if ya did that! _isnt it weird that it only is a .jar file?_

**Comment by TheBigEye** [comment]
**Note:** this is my point of view, ok? it's not an answer 

If the game was an .exe, how would you change the textures ... would you modify it ... or how would you make the game cross-platform?
The game would have to be rewritten, and I mean it, since it was designed to work from a .jar, the textures would not work correctly due to the change of directories within it, a .jar is universal ... you can run in any instead ... of course you need java, but it is easy to install and more secure than an .exe 🤨

Now, do you know how difficult it would be to develop the game? Compile the code, export it to .jar, convert it to .exe, include the JVM inside it (that's because it's still a .jar, just inside it there's an .exe) technically nothing changes by putting a different extension. . it will always be the same code .. only that when including the JVM for its execution there may be errors, or the file may be very heavy 

(Minecraft is not an .exe either, since it uses a Launcher to download and run it, but it is slow, it might be a good idea to make a launcher, I don't know)

**Comment by Makkkkus** [comment]
Launcher idea is good. I think Christoffer has created a launcher.

**Comment by TheBigEye** [comment]
> Launcher idea is good. I think Christoffer has created a launcher.

Although I see it a bit bug, when selecting a version and playing the welcome screen freezes sometimes, of course it is simple and that is good, but it is somewhat outdated

**Comment by Makkkkus** [comment]
Would be cool to create a new launcher

**Comment by thebeeper2524** [comment]
yeah, cuz i dont wanna risk my pc for a PROGRAM.

**Comment by thebeeper2524** [comment]
ok so i just got java rn.

**Comment by thebeeper2524** [comment]
aaand it dosent work.

**Comment by thebeeper2524** [comment]
> Launcher idea is good. I think Christoffer has created a launcher.
which one? theres exactly 1K christoffers on github.

**Comment by Makkkkus** [comment]
https://discord.com/channels/280723930942013440/337449098762387458/364851305480716289
on the discord server:
https://discord.gg/fYAnsuKg7n

**Comment by chrisj42** [comment]
The problem with an exe is that those are OS-specific. Only Windows runs exe files. You can't run those on linux or mac machines. So we would all of a sudden have to release 4-5 different versions of the game every release, and package it all ourselves, every time we wanted to make a release. With java, we just have to make a jar file, and it works everywhere. Java is a very trusted program that is on virtually every computer. So that's most of the reasoning.

**Comment by thebeeper2524** [comment]
I have a windows pc, so its fine.

**Comment by Makkkkus** [comment]
Why do you keep closing and opening this issue?

**Comment by thebeeper2524** [comment]
idk, cuz its fun.

**Comment by thebeeper2524** [comment]
also can someone actually give the launcher a CERTIFICATE and send it, cuz it needs one to actually run.

**Comment by IdotMaster1** [comment]
> also can someone actually give the launcher a CERTIFICATE and send it, cuz it needs one to actually run.

No. It does not need one. Windows is just terrible and thinks that a program without a certificate is a virus or something. I use Linux and forgot how to use Windows lol.

**Comment by Makkkkus** [comment]
No problem!

**Comment by thebeeper2524** [comment]
And my biggest thanks to Oracle, the creators of Java!
